x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/select_direction.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButtonGroup, EuiButtonGroupProps } from '@elastic/eui';
+import React from 'react';
+import { pure } from 'recompose';
+
+import { NetworkTopNFlowDirection } from '../../../../graphql/types';
+
+type MyEuiButtonGroupProps = Pick<
+  EuiButtonGroupProps,
+  'options' | 'idSelected' | 'onChange' | 'color' | 'type'
+> & {
+  name?: string;
+};
+
+const MyEuiButtonGroup: React.SFC<MyEuiButtonGroupProps> = EuiButtonGroup;
+
+const toggleButtonDirection = [
+  {
+    id: NetworkTopNFlowDirection.uniDirectional,
+    label: 'Unidirectional',
```

Comment:
Are these labels suppose to be `i18n` or is this going to be the same labels regardless of locale?